episodes/4-minimal-reproducible-code.Rmd

```diff
@@ -137,7 +145,48 @@ counts_per_day %>%
        y = "Individuals per day",
        fill = "Species")
 
+### ***EDIT-start *** ###
+
+# If I were to try and attempt to plot "krats per day, per plot this is what I would do:
+counts_per_day %>%
+  ggplot(aes(x=plot_type, y=count_per_day, fill=species_id)) +
+  geom_boxplot() 
+
+# what is the group=interaction() adding?
```

Comment:
you're right that this is not necessary. Anyway in the new episode I've simplified this part of the analysis considerably, so it becomes moot!